File: packages/ios/Sources/CivUI/CivInputChip.swift
// CivUI — CivInputChip for SwiftUI
// User-entered token with a remove handle. Mirrors civ-input-chip.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivInputChip: View {
    /// Visible label.
    public var label: String

    /// Identifier passed in the remove-event detail.
    public var value: String

    /// Disabled state.
    public var disabled: Bool

    /// Padding density. `default` | `sm`.
    public var spacing: String

    /// Remove handler.
    public var onRemove: ((String) -> Void)?

    /// Analytics tracker (parallels civ-analytics).
    public var onAnalytics: ((String, [String: Any]?) -> Void)?

    public init(
        label: String = "",
        value: String = "",
        disabled: Bool = false,
        spacing: String = "default",
        onRemove: ((String) -> Void)? = nil,
        onAnalytics: ((String, [String: Any]?) -> Void)? = nil
    ) {
        self.label = label
        self.value = value
        self.disabled = disabled
        self.spacing = spacing
        self.onRemove = onRemove
        self.onAnalytics = onAnalytics
    }

    public var body: some View {
        EmptyView()
    }
}

File: packages/android/src/main/kotlin/gov/civui/components/CivInputChip.kt
// CivUI — CivInputChip for Jetpack Compose
// User-entered token with a remove handle. Mirrors civ-input-chip.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivInputChip(
    label: String = "",
    value: String = "",
    disabled: Boolean = false,
    spacing: String = "default",
    modifier: Modifier = Modifier,
    onRemove: ((String) -> Unit)? = null,
    onAnalytics: ((event: String, data: Map<String, Any>?) -> Unit)? = null,
) {
    Box(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
    }
}
